web/directory/directory/views/index_view.py

```diff
@@ -1,9 +1,48 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+
+import requests
 from django.views.generic import TemplateView
 
+from django.conf import settings
+
+ORGANIZATION_LIST_VIEW_URL_PATH = "api/v1/organization/"
+DEFAULT_LIMIT = "50"
+
 
 class IndexView(TemplateView):
     view_name = "home_page_view"
     template_name = "base.html"
 
     def get_context_data(self, **kwargs):
-        return {}
+        name = self.request.GET.get("name")
+        if not name:
+            return {"results": []}
+        data = filter_organizations_by_name(name)
+        return {"results": data["results"], "current": name}
```

Comment:
This should do a bit more error checking.  When I didn't have the token configured correctly this threw a 500 since data didn't have a "results".  The data did have the error message in it, so it would have been better if the page just showed the error message (or more generally the contents of `data`) if there are no results.